mbuild/compound.py

```diff
@@ -2174,6 +2174,51 @@ def spin(self, theta, around):
         self.rotate(theta, around)
         self.translate(center_pos)
 
+    # Interface to GMSO Topology for reading/writing mol2 files
+    def from_gmso(self, topology, coords_only=False, infer_hierarchy=True):
+        """Convert a GMSO Topology to mBuild Compound.
+
+        Parameter
+        ---------
+        topology : gmso.Topology
+            The GMSO Topology to be converted.
+        compound : mb.Compound, optional, default=None
+            Host mb.Compound that we are loading to.
+        coords_only : bool, optional, default=False
+            Set preexisting atoms in compound to coordinates given by Topology.
+        infer_hierarchy : bool, optional, default=True
+            If True, infer compound hierarchy from Topology residue, to be implemented.
+
+        Returns
+        -------
+        compound : mb.Compound
+        """
+        from mbuild.conversion import from_gmso
```

Comment:
We already import `conversion` at the top of this file. I think

```py3
return conversion.from_gmso(...)
```

I think that also makes it a bit easier to read as well.